packages/react-scripts/config/webpack.development.js

```diff
@@ -0,0 +1,62 @@
+// @remove-on-eject-begin
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @remove-on-eject-end
+'use strict';
+
+const path = require('path');
+const paths = require('./paths');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
+const WatchMissingNodeModulesPlugin = require('react-dev-utils/WatchMissingNodeModulesPlugin');
+
+// We are in the development config. This contains development only features.
+module.exports = function() {
+  return {
+    devtool: 'cheap-module-source-map',
+    // Note: instead of the default WebpackDevServer client, we use a custom one
+    // to bring better experience for Create React App users. You can replace
+    // the line below with these two lines if you prefer the stock client:
+    // require.resolve('webpack-dev-server/client') + '?/',
+    // require.resolve('webpack/hot/dev-server'),
+    entry: [
+      require.resolve('react-dev-utils/webpackHotDevClient'),
+      paths.appIndexJs,
+    ],
+    output: {
+      pathinfo: true,
+      filename: 'static/js/bundle.js',
+      chunkFilename: 'static/js/[name].chunk.js',
+      devtoolModuleFilenameTemplate: info =>
+        path.resolve(info.absoluteResourcePath).replace(/\\/g, '/'),
+    },
+    plugins: [
+      new HtmlWebpackPlugin(
+        Object.assign(

```

Comment:
Is this `Object.assign` necessary?

Reply:
If they are separated now I don't think so. Looks like it was more necessary when they were combined with the ternary.